src/App.jsx

```diff
@@ -7,7 +10,15 @@ function App() {
   const { user, recruiter } = useAuth();
```

Comment:
If don't use the variable `recruiter` you can add an underscore to say that if declared but don't use it yet

```suggestion
  const { user, _recruiter } = useAuth();
```